src/coreml.cs

```diff
@@ -1097,4 +1133,16 @@ interface MLModelCollectionEntry {
 		[Export ("isEqualToModelCollectionEntry:")]
 		bool IsEqual (MLModelCollectionEntry entry);
 	}
+
+	[Watch (9,0), TV (16,0), Mac (13,0), iOS (16,0), MacCatalyst (16,0)]
```

Comment:
`Watch (9,0)` ? The type seems to be useless, the only method seems to be "NoWatch"

Reply:
Seems to be needed for loadModelAsset:configuration:completionHandler: which is present on watchOS. The model asset can be loaded form a file. Maybe there is an API we are missing about that.